webpack.renderer.config.js

```diff
@@ -102,22 +113,42 @@ const rendererConfig = {
       govern: resolve(`app/src/renderer/components/govern`),
       staking: resolve(`app/src/renderer/components/staking`),
       wallet: resolve(`app/src/renderer/components/wallet`)
+      // moment: `moment/src/moment`, // this will remove locales, we rely on them
```

Comment:
do we need these comments?

Reply:
Was the alias system for reduce the size of moment and lodash, but we are using their inner dependencies so we can't optimize for now..I'll remove them